src/tests/functional/shared_test_classes/src/base/low_precision_transformations/layer_transformation.cpp

```diff
@@ -87,4 +76,108 @@ std::string LayerTransformation::getTestCaseNameByParams(
+    const std::shared_ptr<const ov::Model>& function = execNet.get_runtime_model();
+    const auto& execFunction = function;
+
+    for (const auto& op : execFunction->get_ops()) {
+        const auto& rtInfo = op->get_rt_info();
+        const auto& typeIt = rtInfo.find("layerType");
+
+        OPENVINO_ASSERT(typeIt != rtInfo.end(), "Layer is not found for type: ", layerType);
+
+        auto type = typeIt->second.as<std::string>();
+        if (type == layerType) {
+            const auto& it = rtInfo.find("runtimePrecision");
+            OPENVINO_ASSERT(it != rtInfo.end(), "Runtime precision is not found for node: ", type);
+            return it->second.as<std::string>();
+        }
+    }
+
+    return "";
+}
+
+std::string LayerTransformation::getRuntimePrecisionByFusedName(const std::string& layerName) {
+    const ov::CompiledModel& execNet = compiledModel;
+    const std::shared_ptr<const ov::Model>& function = execNet.get_runtime_model();
+    const auto& execFunction = function;
+
+    const auto parse = [](const std::string& originalLayersNames) -> std::set<std::string> {
+        std::set<std::string> names;
+
+        std::string tmp = originalLayersNames;
+        size_t beginPosition = 0ul;
+        size_t endPosition;
+        while ((endPosition = tmp.find(",", beginPosition)) != std::string::npos) {
+            names.insert(tmp.substr(beginPosition, endPosition - beginPosition));
+            beginPosition = endPosition + 1;
+        }
+
+        names.insert(tmp.substr(beginPosition, endPosition - beginPosition));
+        return names;
+    };
+
+    for (const auto& op : execFunction->get_ops()) {
+        const auto& rtInfo = op->get_rt_info();
+
+        const auto& nameIt = rtInfo.find("originalLayersNames");
+        OPENVINO_ASSERT(nameIt != rtInfo.end(), "originalLayersNames is not found for node: ", layerName);
+        const auto fusedName = parse(nameIt->second.as<std::string>());
+        if (fusedName.find(layerName) == fusedName.end()) {
+            continue;
+        }
+
+        const auto& it = rtInfo.find("runtimePrecision");
+        OPENVINO_ASSERT(it != rtInfo.end(), "runtimePrecision is not found for node: ", layerName);
+        const auto rtPrecisionPtr = it->second.as<std::string>();
+        return rtPrecisionPtr;
+    }
+
+    return "";
+}
+
+std::map<std::string, ngraph::Node::RTMap> LayerTransformation::getRuntimeInfo() {
+    const ov::CompiledModel& execNet = compiledModel;
+    const std::shared_ptr<const ov::Model>& function = execNet.get_runtime_model();
+
+    std::map<std::string, ngraph::Node::RTMap> runtimeInfo;
+    for (const auto& op : function->get_ops()) {
+        runtimeInfo[op->get_friendly_name()] = op->get_rt_info();
+    }
+    return runtimeInfo;
+}
+
+void LayerTransformation::init_input_shapes(const ov::PartialShape& shape) {
+    std::pair<ov::PartialShape, std::vector<ov::Shape>> input_shapes(shape, { shape.to_shape() });
```

Comment:
Use `static_partial_shapes_to_test_representation()` instead of 2 lines? File: `ov_subgraph.hpp`

Reply:
I did not get any response to the comment. Why should it be resolved?